### PR TITLE
Run test on random region (except for ec2)

### PR DESCRIPTION
**Pull request by aguillenv**
Ignoring EC2 due to the special casing with AMIs.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
   push:
     branches: [main]
   pull_request:
-    branches: [main]
+    branches: []
```
Comment (aguillenv) [inline .github/workflows/ci.yml:11]: Temporal! just to avoid test suite interference right now

Comment (dfellis) [reply .github/workflows/ci.yml:11]: It's running anyways :(

File: test/modules/aws-account-integration.ts [hunk 2]
```diff
@@ -21,7 +22,7 @@ describe('AwsAccount Integration Testing', () => {
   // TODO: Restore some mechanism to verify credentials
   /*it('does not create a test DB with fake credentials', (done) => void iasql.add(
     dbAlias,
-    'us-west-2',
+    getRandomRegion(),
```
Comment (dfellis) [inline test/modules/aws-account-integration.ts:25]: You need to do this at the top of the test one time, or it'll do different regions for the different sets of tests and could cause failures

Comment (dfellis) [reply test/modules/aws-account-integration.ts:25]: Just for this test, afaict

**Review by depombo — APPROVED** [review]

**Review by depombo — APPROVED** [review]

**Pull request merged**